### ISSUE TITLE
update for 1.3.1

### ISSUE THREAD

**Issue by raidernick** [KSP-RO/RSSTimeFormatter#8]
@jwvanderbeck can we get a 1.3.1 release, this is one of the things holding up the RO release.

**Comment by jwvanderbeck** [comment]
I'm not setup any more for compiling these.  What specifically is holding things up?

**Comment by raidernick** [comment]
well this is a dependency of RO on the forum as well as ckan, ckan won't index RO if the dependencies aren't all up to date

**Comment by jwvanderbeck** [comment]
No, I mean what is preventing anyone else with RO GitHub access from compiling this and pushing an update? Is there something needed from me?

**Comment by raidernick** [comment]
nobody has access to make updates here but you and nk...? The git access for users is totally screwed.

**Comment by jwvanderbeck** [comment]
I think I have that fixed now
